compile.sh

```diff
@@ -5,3 +5,4 @@ RUSTFLAGS="-Zlocation-detail=none" cargo +nightly build --manifest-path rust/Car
 kotlinc -include-runtime kotlin/code.kt -d kotlin/code.jar
 #kotlinc-native -include-runtime kotlin/code.kt -d kotlin/code
 dart compile exe dart/code.dart -o dart/code --target-os=macos
+swiftc -O swift/code.swift -o swift/code
```

Comment:
`-Ounchecked` speeds this up a fair bit vs just `-O`, but I'm not sure if it's a fair comparison to other languages. It removes force nil unwrap checking, array out of bounds checking, etc